src/Agent.ts

```diff
@@ -32,8 +37,7 @@ export class Agent {
   private _onErrorFns = [];
 
   constructor(options: TrackJSInstallOptions) {
-    this.options = Object.assign({}, Agent.defaults, options);
-    this.options.correlationId = this.options.correlationId;
```

Comment:
What's the story with this guy? `this.options.correlationId = this.options.correlationId;`

Reply:
haha yea, that was vestigial from this change:
https://github.com/TrackJs/TrackJS-Node/commit/1a9e081f324dea58b2914dfef9e36e2f261c5a2e

We used to always set it to a uuid, but ditched that so its only what the user provides ... and its redundant given the line before it.